### PR TITLE
Add TWZRD Agent Intel MCP server to Tools section

### PR DESCRIPTION
**Pull request by twzrd-sol**
## Summary

Adds [TWZRD Agent Intel](https://intel.twzrd.xyz) to the **Tools & Software** section.

When running multi-agent fine-tuning pipelines (multiple agents consuming reward data, accessing annotation APIs, or making micropayments), TWZRD Agent Intel lets you verify agent wallet identity and trust scores on Solana before granting access. It's especially relevant for teams using autonomous agents to collect or label training data.

**Free MCP config:**
```json
{"mcpServers":{"twzrd-agent-intel":{"url":"https://intel.twzrd.xyz/mcp"}}}
```

**Free tools:** `score_agent(wallet)`, `preflight_check(wallet)`

### CODE REVIEW

**Comment by twzrd-sol** [comment]
Withdrawing this PR — it was off-topic for this list and was opened by an over-broad automated submission process on our side. Apologies for the noise.
